README.md

```diff
@@ -39,12 +39,6 @@ python generate_vocab.py --mode diagnosis --add-terms ./openneuro_diagnoses.tsv
 
 ## To compare the old and new version of a vocabulary term list
 
-1. Create the following subdirectories:
-   ```bash
-   mkdir vocab/old vocab/new
-   ```
-2. Copy your old and new vocabulary list .json file into the appropriate subdirectory.
-3. Run:
-   ```bash
-   python src/compare_vocabs.py
-   ```
\ No newline at end of file
+```bash
+python src/compare_vocabs.py --old-terms PATH/TO/OLD/VOCAB/JSON --new-terms PATH/TO/NEW/VOCAB/JSON
```

Comment:
The makefile already contains specific commands for the different vocabularies (i.e. `make diagnosis` and `make assessment`) - you could think of adding the `compare` thing in there because it'll always be a comparison within the same type of vocabulary

Reply:
I'll leave this for a future enhancement, since atm I like having the flexibility to specify the paths to the JSONs (since I tend to rename them locally each time so I remember the versions) - it seems like having arguments is not the easiest in makefiles?